src/main/scala/mesosphere/marathon/storage/migration/MigrationTo19100.scala

```diff
@@ -240,7 +240,7 @@ object MigrationTo19100 extends MaybeStore with StrictLogging {
           case Some(rootGroupVersion) => zkStore.get(RootId, rootGroupVersion).map(group => (group, Some(rootGroupVersion)))
           case None => zkStore.get(RootId).map(group => (group, None))
         }
-        .collect{ case (Some(group), optVersion) if group.enforceRole.isEmpty => (group, optVersion) }
+        .collect{ case (Some(group), optVersion) => (group, optVersion) }
```

Comment:
can you explain how you discovered this was the issue? it seems like empty enforceRole was the issue?

Reply:
So, the protobuf conversion set some arbitrary `enforceRole`. This would then filter all groups.